code/index.html

```diff
@@ -1,30 +1,140 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>My business webiste</title>
+  <link rel='stylesheet' href='https://unpkg.com/emoji.css/dist/emoji.min.css'>
+  <link rel="stylesheet" href="style.css">
+  <link rel="shortcut icon" href="./Images/logoproject3.jpg">
+
+</head>
+
+<body>
+
+  <header class="navbar">
+    <ul class="navbar-content">
+      <img class="icon-main" src="./Images/logoproject3.jpg" alt="icon-main">
```

Comment:
The alt text should be written as "normal" text so no need to use `-` between words!
```suggestion
      <img class="icon-main" src="./Images/logoproject3.jpg" alt="logotype">
```